include/functions.php

```diff
@@ -26,6 +26,33 @@ function wpcom_vip_get_page_by_path( $page_path, $output = OBJECT, $post_type =
 	}
 }
 
+if ( ! function_exists( 'sanitize_locale_name' ) ) {
+	/**
+	 * Strips out all characters not allowed in a locale code.
+	 *
+	 * @since 3.5
+	 * @since WP 6.2.1
```

Comment:
I'd like to add the backward compatibility comment that I usually serch for when I remove old code.
Also, I propose to keep `@since` tag for our own versions.
```suggestion
	 * Strips out all characters not allowed in a locale code.
	 *
	 * Backward compatibility with WP < 6.2.1.
	 *
	 * @since 3.5
```

Reply:
Done in [92f0b2f](https://github.com/polylang/polylang/pull/1280/commits/92f0b2f013f5527be165cb8f0c1cf3bd73e2f458).